### PR TITLE
adding disarium number check and infix to postfix using stack

### PR DESCRIPTION
**Pull request by ashish4321**
adding disarium number check and infix to postfix using stack

### CODE REVIEW

File: disarium.c [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#include <stdio.h>
+#include<math.h>
+
```
Comment (dynamitechetan) [inline disarium.c]: can we have some comments to increase readability?

File: convert.c [hunk 2]
```diff
@@ -0,0 +1,57 @@
+#include<stdio.h>
```
Comment (dynamitechetan) [inline convert.c]: some comments or docs please

File: disarium.c [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+int main() 
+{
+ int n,i=0,j,sum=0,l,k;
+ scanf("%d",&n);
+ k=n;
+ j=k;
+ while(n>0)
+ {
+  d=n%10;
+ n=n/10;
```
Comment (dynamitechetan) [inline disarium.c]: fix indentation

File: convert.c [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        else if(*e == ')')
+        {
+            while((x = pop()) != '(')
+                printf("%c", x);
+        }
+        else
+        {
+            while(priority(stack[top]) >= priority(*e))
+                printf("%c",pop());
+            push(*e);
```
Comment (dynamitechetan) [inline convert.c]: indentation issues

File: convert.c (hunk 2 above)
Comment (ashish4321) [reply convert.c]: i will add some comments

**Pull request merged**